package.json

```diff
@@ -22,9 +23,12 @@
     "babel-loader": "^7.1.2",
     "babel-minify-webpack-plugin": "^0.2.0",
     "babel-preset-env": "^1.6.1",
-    "css-loader": "^0.28.7",
-    "file-loader": "^1.1.5",
+    "clean-webpack-plugin": "^0.1.18",
+    "css-loader": "^0.28.10",
+    "file-loader": "^1.1.11",
     "style-loader": "^0.19.0",
-    "webpack": "3.8.1"
+    "webpack": "^4.0.1",
```

Comment:
Do the babel things need to be updated along with this? I tried running yarn and saw these warnings:

warning " > babel-loader@7.1.2" has incorrect peer dependency "webpack@2 || 3".
warning " > babel-minify-webpack-plugin@0.2.0" has incorrect peer dependency "webpack@^2.0.0 || ^3.0.0".